wkcuber/convert_nifti.py

```diff
@@ -111,18 +113,17 @@ def to_target_datatype(
 
 
 def convert_nifti(
-    source_nifti_path,
-    target_path,
-    layer_name,
-    dtype,
-    scale,
-    mag=1,
-    file_len=DEFAULT_WKW_FILE_LEN,
-    bbox_to_enforce=None,
-    write_tiff=False,
-    use_orientation_header=False,
-    flip_axes=None,
-):
```

Comment:
The parameter `mag` is never used (it gets overwritten). Therefore, I removed the parameter. Is that correct?

Reply:
Seems fine for me, let's just check who might call this in any dependent projects.